src/Marketplace/View.php

```diff
@@ -455,6 +456,16 @@ public static function displayList(
             $networkmail  = GLPI_NETWORK_MAIL;
             $refresh_lbl  = __("Refresh plugin list");
             $search_label = __("Filter plugin list");
+            $url = "'" . Plugin::getFormURL() . "'";
+            $fields = "{ 'formIdentifier' : 'change_default_interfaceform', '_glpi_csrf_token' : '" . Session::getNewCSRFToken() . "', 'value' : 2}";
+            $default = $displaybutton = '';
+            if (Config::getConfigurationValue('core', 'marketplace_replace_plugins') == 2) {
+                $default = 'checked disabled';
+            } else if (Config::getConfigurationValue('core', 'marketplace_replace_plugins') == 1) {
+                $displaybutton = 'd-none';
+            }
```

Comment:
```suggestion
 $marketplace_replace_plugins = Config::getConfigurationValue('core', 'marketplace_replace_plugins');
            if ($marketplace_replace_plugins == Controller::MP_REPLACE_YES) {
                $default = 'checked disabled';
            } else if ($marketplace_replace_plugins == Controller::MP_REPLACE_ASK || is_null($marketplace_replace_plugins)) {
                $displaybutton = 'd-none';
            }
```

Reply:
Use the constant to check value

e.g : ```Controller::MP_REPLACE_YES```

And check for ```null``` value from fresh install